web/react/components/filtered_user_list.jsx

```diff
@@ -48,6 +92,7 @@ class FilteredUserList extends React.Component {
 
         let users = this.props.users;
 
+        // this logic might be better placed in UserStore
```

Comment:
It's not that we should store the filtered users, but that it should be asking the store for the filtered users. The store does the logic on it's stored list of all users then returns the list of filtered users. Basically, the way I see it components are supposed to be as dumb as possible and just render the stuff they're given without manipulation.

That said, it doesn't really matter in this case since the users are passed in as a prop so it wouldn't make much sense to ask the UserStore for a new list of users when we already have the users.

0/5 on this, so I'll remove the comment